stacks/java-openliberty/devfile.yaml

```diff
@@ -14,24 +14,30 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
+
 schemaVersion: 2.1.0
 metadata:
   name: java-openliberty
-  version: 0.8.1
-  displayName: 'Open Liberty Maven'
+  version: 0.8.2
```

Comment:
What would be the downside of leaving the version as-is for the four Liberty (Open/WebSphere , Maven/Gradle) devfiles?

We started down a path of using our own versioning strategy integrating our outer loop artifacts.   Following that to its conclusion we'd want to make other changes.

But if this change is just syntax cleanup, well, I think we could say this is equivalent to our previous version, e.g. 0.8.1 in this case, and merge in the change.   

(And we could open an issue in our devfile source repo to make the same changes there:  https://github.com/OpenLiberty/devfile-stack/issues ).   WDYT?

Reply:
In general, we recommend version updates (semver) when the stack is updated so that users know the underlying stack has been changed. Having said that, it is just a recommendation so it is up to the stack owner to have the final decision on the version number.

Reply:
I'd rather just leave the version alone in this PR then.

I opened: https://github.com/OpenLiberty/devfile-stack/issues/212 to follow-up and make these changes.  We can still view that repo as the source of record of our devfile.

But I think it's OK to go ahead and merge the rest of the changes.  We might not get to it for a few weeks so don't want to keep you waiting.   

There's an opportunity for minor user confusion here, I know, but unless there's some specific reason (like some version of some devfile tooling having a bug in the older syntax) then I don't think it will end up mattering all that much.